.pipelines/templates/aks-setup.yaml

```diff
@@ -8,12 +13,22 @@ steps:
 
       az group create -n ${AZURE_CLUSTER_NAME} -l ${AKS_CLUSTER_LOCATION} > /dev/null 2>&1
 
-      az aks create \
-          -g ${AZURE_CLUSTER_NAME} \
-          -n ${AZURE_CLUSTER_NAME} \
-          --enable-managed-identity \
-          --no-ssh-key \
-          > /dev/null 2>&1
+      if [[ "${{ parameters.aksAddonRun }}" == "True" ]]; then
+        az aks create \
+            -g ${AZURE_CLUSTER_NAME} \
```

Comment:
why is the rg same as clutter name?

Reply:
This is the format used in the aks-setup in Upstream ADO. I think because both the cluster and resource group get created at the same time (and is a randomly generated name), and also get deleted together after the job runs, the same name could be used for both.